archetypes/gwtp-appengine-guice/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.arcbees.archetypes</groupId>
+    <artifactId>gwtp-appengine-guice-archetype</artifactId>
+    <version>1.1-SNAPSHOT</version>
```

Comment:
I think we should follow GWTP versioning here as well (should also applies to other POMs).